### PR TITLE
Change toolbar colour to blue

### LINKED ISSUE #1761: Change toolbar colour to blue
Change toolbar background colour to blue #2196f3
- [x] Start view
- [x] Edit view
- [x] Projects view
- [x] New project view
- [x] Tags view
- [x] Clients view
- [x] Also, change text colour and icons to white

Zeplin (Start view example): https://zpl.io/bzy1R3z

### PR DESCRIPTION
**Pull request by daividssilverio**
Closes #1761 

Update toolbars for the Start view, Edit view, Projects view,  New project view, Tags view and Clients view with the new blue toolbar colour.

squash whenever

### CODE REVIEW

File: Toggl.Giskard/Activities/EditTimeEntryActivity.cs [hunk 1]
```diff
@@ -18,6 +21,8 @@ public sealed class EditTimeEntryActivity : MvxAppCompatActivity<EditTimeEntryVi
     {
         protected override void OnCreate(Bundle bundle)
         {
+            this.ChangeStatusBarColor(new Color(ContextCompat.GetColor(this, Resource.Color.blueStatusBarBackground)), false);
```
Comment (willsb) [inline Toggl.Giskard/Activities/EditTimeEntryActivity.cs]: You can omit the parameter when false is passed

**Comment by daividssilverio** [comment]
@willsb it works fine on lollipop! As you said earlier, what shouldn't work are the status bar icons colour change.

screen shot 2018-03-20 at 7 58 26 pm

File: Toggl.Giskard/Activities/EditTimeEntryActivity.cs (hunk 1 above)
Comment (daividssilverio) [reply Toggl.Giskard/Activities/EditTimeEntryActivity.cs]: oops, didn't see the default value, my bad.

Comment (kornelijepetak) [reply Toggl.Giskard/Activities/EditTimeEntryActivity.cs]: That's an argument, not a parameter, @willsb 😛

**Pull request merged**